resources/views/themes/default1/front/checkout.blade.php

```diff
@@ -434,4 +434,52 @@
     
 </div>
 @endif
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.7.1/jquery.min.js" type="text/javascript"></script>
+<script>
+  $('#checkoutsubmitform').submit(function(){
+     $("#proceed").html("<i class='fa fa-circle-o-notch fa-spin fa-1x fa-fw'></i>Please Wait...")
+    $("#proceed").prop('disabled', true);
+
+  });
+     $(document).ready(function(){
+            var finalPrice = $('#total-price').val();
+            console.log(finalPrice);
+        $("#rzp_selected").click(function(){
+            
+            var processingFee = $(this).attr('data-currency');
+            var totalPrice = finalPrice;
+            $('#fee').hide();
+            $.ajax({
+                type:'POST',
+                data: {'processing_fee':processingFee,'price':totalPrice},
+                 beforeSend: function () {
+                 $('#response').html( "<img id='blur-bg' class='backgroundfadein' style='width: 50px; height:50 px; display: block; position:    fixed;' src='{!! asset('lb-faveo/media/images/gifloader3.gif') !!}'>");
+                },
+                 url: "{{url('update-final-price')}}",
+                 success: function () {
+              }
+            });
+        }); 
+        $("#allow_gateway").click(function(){
+            var processingFee = $(this).attr('data-currency');
+            var totalPrice = finalPrice;
+            $('#fee').show();
+            $.ajax({
+                type:'POST',
+                data: {'processing_fee':processingFee,'price':totalPrice},
+                 beforeSend: function () {
+                 $('#response').html( "<img id='blur-bg' class='backgroundfadein' style='width: 50px; height:50 px; display: block; position:    fixed;' src='{!! asset('lb-faveo/media/images/gifloader3.gif') !!}'>");
+                },
+                 url: "{{url('update-final-price')}}",
+                 success: function () {
```

Comment:
remove this if we are not handling success or failure

Reply:
Fixed